confgenerator/fluentbit/conf_test.go

```diff
@@ -296,9 +296,9 @@ func TestTail(t *testing.T) {
 			expectedTailConfig: `[INPUT]
```

Comment:
Btw, I would like to get rid of all these unit tests and just use the file-based tests.

Reply:
Yep, future PRs.